src/themes/masschallenge/modules/modal.variables

```diff
@@ -0,0 +1,9 @@
+/********** colors **********/
+
+@textColor: @itemTextColor;
+@textareaBackground: #F1F3F4;
+@textareaColor: #8D8F94;
```

Comment:
Lowercase to match the existing style unless we are deiding to use uppercase from now on.